doc/build-versioned-docs.sh

```diff
@@ -22,11 +22,29 @@ languages="en `find locales/ -mindepth 1 -maxdepth 1 -type d -exec basename '{}'
 # Get a list of tags, excluding those before v1.0.0
 versions="`git for-each-ref '--format=%(refname:lstrip=-1)' refs/tags/ | grep -iE '^v((([1-9]|[0-9]{2,}).*\.([8-9]|[0-9]{2,}).*)|([2-9]|[0-9]{2,}).*)$'`"
 
+# Set the numpy version to use for v1.8.0 to v1.12.0
+numpy_version_1="1.26.4"
+numpy_version_2=$(python -c "import numpy; print(numpy.__version__)")
+
 for current_version in ${versions}; do
  
   # Make the current language available to conf.py
   export current_version
   git checkout --force ${current_version}
+
+  # Downgrade numpy for versions between v1.8.0 and v1.12.0 to avoid conflicts in docs
+  if [ "$current_version" = "v1.8.0"  ] || \
+     [ "$current_version" = "v1.9.0"  ] || \
+     [ "$current_version" = "v1.10.0" ] || \
+     [ "$current_version" = "v1.11.0" ] || \
+     [ "$current_version" = "v1.11.1" ] || \
+     [ "$current_version" = "v1.12.0" ]; then
```

Comment:
```suggestion
  if echo "$current_version" | grep -qE '^v1\.(8\.0|9\.0|10\.0|11\.0|11\.1|12\.0)$'; then
```
I think we could try something like that

Reply:
But it's only a cosmetic change, so I will approve the PR